src/plugins/maps_legacy/config.ts

```diff
@@ -29,7 +29,7 @@ export const configSchema = schema.object({
   manifestServiceUrl: schema.string({ defaultValue: '' }),
   emsFileApiUrl: schema.string({ defaultValue: 'https://vector.maps.elastic.co' }),
   emsTileApiUrl: schema.string({ defaultValue: 'https://tiles.maps.elastic.co' }),
-  emsLandingPageUrl: schema.string({ defaultValue: 'https://maps.elastic.co/v7.7' }),
+  emsLandingPageUrl: schema.string({ defaultValue: 'https://maps.elastic.co/v7.10' }),
```

Comment:
Interesting, this is so old. I guess the config location moved since https://github.com/elastic/kibana/pull/71532/files

Reply:
😱 thx for pointing out